vllm/v1/request.py

```diff
@@ -118,9 +118,12 @@ def __init__(
         # indicates that the output is corrupted
         self.num_nans_in_logits = 0
 
-        # The number of requests being preempted by the scheduler
+        # The number of requests being preempted by the scheduler.
```

Comment:
Completely unrelated to this PR, but since you modified the line ... :rofl: 

This is actually the number of times this request was preempted?